routes/wishlist.router.js

```diff
@@ -0,0 +1,40 @@
+const express = require("express");
+const router = express.Router();
+const Product = require("../model/products.model");
+
+router.get("/", (req, res) => {
+	Product.find({ wishListed: true })
+		.select("-__v")
+		.then((data) => res.json({ items: data }))
+		.catch((error) => console.error(error));
+});
```

Comment:
Try using async await way instead of .then

Reply:
done